core/includes/color.inc

```diff
@@ -47,7 +47,7 @@ class Color {
     // Ignore '#' prefixes.
     $hex = ltrim($hex, '#');
 
-    // Convert shorhands like '#abc' to '#aabbcc'.
+    // Convert shorthand colors, for example '#abc', to '#aabbcc'.
```

Comment:
@BWPanda I believe that @kiamlaluno meant this:
```suggestion
    // Convert shorthand colors, for example '#abc' to '#aabbcc'.
```

Reply:
...or something this?:
```suggestion
    // Convert shorthand colors (for example '#abc' to '#aabbcc').
```

Reply:
I decided to use the word 'expand' instead of convert, because then it still makes sense if you remove the example.